las/__init__.py

```diff
@@ -49,12 +49,17 @@ def g():
 
         try:
             execution = las_client.get_transition_execution(transition_id, execution_id=execution_id)
-            output = f(las_client, execution['input'])
+            result = f(las_client, execution['input'])
+            
+            try:
+                output, status = result
+            except ValueError:
+                output, status = result, 'succeeded'
             
             las_client.update_transition_execution(
                 transition_id=transition_id,
                 execution_id=execution_id,
-                status='succeeded',
+                status=status,
                 output=output,
```

Comment:
If status is 'failed', error is a required argument. So I believe this will throw an exception if status from the result is not 'succeeded'